README.md

```diff
@@ -196,6 +196,13 @@ next_url = page1.next_page_url
 page2 = shopify.Product.find(from_=next_url)
 ```
 
+## Set up pre-commit locally [OPTIONAL]
+[Pre-commit](https://pre-commit.com/) is set up as a GitHub action that runs on pull requests and pushes to the `master` branch. If you want to run pre-commit locally, install it and set up the git hook scripts
```

Comment:
Does this mean there will be commits on master that will show up as being "authored" by Pre-commit?

Reply:
No, sorry I meant it runs on "pull requests" and "pushes" to the `master` branch. So since the master branch is protected now, it basically only needs to run on "pull requests". When it just checks the code, it will tell you which lines are failing pre-commit and what you should change. It can also change code but it will show the author as the person who initialized the commit

Reply:
hey @andyw8 ! I just tested out the pre-commit GitHub Action. If you look at one of the failing builds like [this one](https://github.com/Shopify/shopify_python_api/runs/1921696279#step:4:51) you'll see `files were modified by this hook`. Re-writing the code is normally done by pre-commit when installed and run locally, but since it's a GitHub Action, it only fails the CI and does not change anything. The user is then prompted to make the change either manually or by installing pre-commit.

Reply:
Also, follow-up to your initial question: when you have pre-commit installed locally, you'll do 
```
git add .
git commit -m "Some commit message here"
```
Once you run the commit, pre-commit scans the files and modifies them as needed. You'll have your staged changes plus the new unstaged changes created by pre-commit. You'll need to do
```
git add .
git commit -m "Some commit message"
```
again to re-try the commit. If you pass pre-commit, you will be able to do `git push`.

![image](https://user-images.githubusercontent.com/40904028/108262566-171d5080-7133-11eb-8021-6d6d1dc65558.png)
The end result is a singe change that has been verified by pre-commit but authored by you.